web/locales/de.json

```diff
@@ -93,7 +93,7 @@
         "consumption": "Verbrauch",
         "cpinfo": "<b>Verbrauch</b> bezieht Importe und Exporte mit ein, <b>Produktion</b> ignoriert sie",
         "selectLanguage": "Sprache auswählen",
-        "lowCarbDescription": "Beinhaltet regenerative Energie and Atomkraft"
+        "lowCarbDescription": "Beinhaltet regenerative Energie and Kernenergie"
```

Comment:
two old typos could be fixed in the same run:
"Beinhaltet regenerative Energie***n***  ***u***nd Kernenergie"